### ISSUE TITLE
Can't use/cache feedlint (Download failed: Downloaded archive has incorrect size.)

### ISSUE THREAD

**Issue by Abscissa** [0install/feedlint#1]
```
% 0install add feedlint http://0install.net/2007/interfaces/FeedLint.xml
URL: https://downloads.sourceforge.net/project/zero-install/0install/2.3.5/0install-2.3.5.tar.bz2
Expected: 466037 bytes
Received: 377 bytes
... downloading implementation http://0install.net/2007/interfaces/ZeroInstall.xml 2.3.5 (id=sha1new=6f4892dfcb947a03dd3b89f04b2d0650e871a060)
Downloaded archive has incorrect size.
URL: https://downloads.sourceforge.net/project/zero-install/FeedLint/0.9/feedlint-0.9.tar.bz2
Expected: 11770 bytes
Received: 355 bytes
... downloading implementation http://0install.net/2007/interfaces/FeedLint.xml 0.9 (id=sha1new=b6774ab624a628e7d18c3c9218587db6c9112262)
```

**Comment by talex5** [comment]
Is this the same thing you reported in https://github.com/0install/0install/pull/107? If so, it should have been fixed by 0install 2.14.1, so make sure you're using that version - it works for me.

**Comment by Abscissa** [comment]
Oops, you're right of course. That's what I get for trying to code too late into the night!
